src/core/layout/scrollable-list/scrollable-list.js

```diff
@@ -122,16 +123,19 @@ const updatePanelList = panel => {
     const tableContainer = panel.getByName("gridContainer", true);
     const scene = panel.parentContainer.scene;
     tableContainer.clear(true);
-    tableContainer.add(createTable(scene, panel.name, panel.callback));
+    tableContainer.add(createTable(scene, panel.name, panel.callback, panel.parentContainer));
```

Comment:
Thinking about this - could you just pass panel in? (I realise this isn't exactly part of this exact PR... :)

Reply:
`createTable` gets called at construct time and exists before `panel` does in that context. Suppose I could add a wrapping fn that destructures the panel and calls `createTable` but I don't think it'd be any clearer...